### PR TITLE
Info for virtual env install. Years 2017 to 2022 in output.

### PR DESCRIPTION
**Pull request by ModelEarth**
2019 was already in output and did not change when rerunning.

### CODE REVIEW

File: import_factors_exio/README.md [hunk 1]
```diff
@@ -1,5 +1,71 @@
...
+    ## Checking python bits for USEEIO install
+    #import platform
+    ## Either
+    #st.write(platform.architecture())
+    #print(platform.architecture())
+-->
+
+Install esupy and fedelemflowlist
+
+	pip install git+https://github.com/USEPA/esupy.git#egg=esupy
```
Comment (bl-young) [inline import_factors_exio/README.md:40]: esupy is a requirement of fedelemflowlist, so no need to install it separately

File: import_factors_exio/README.md [hunk 2]
```diff
@@ -1,5 +1,71 @@
...
+If git-lfs not found next time, run the above outsite virtual env.
+Was able to ignore the following the first time:
+warning: current user is not root/admin, system install is likely to fail.
+warning: error running /Applications/Xcode.app/Contents/Developer/usr/libexec/git-core/git 'config' '--includes' '--system' '--replace-all' 'filter.lfs.clean' 'git-lfs clean -- %f': 'error: could not lock config file /etc/gitconfig: Permission denied' 'exit status 255'
+
+
+[The wiki](https://github.com/USEPA/fedelemflowlist/wiki/Install#installation-of-python-module-and-dependencies) includes installation instructions for fedelemflowlist. Make sure to use the latest release and not v1.0.8, or just put nothing at all.
+-->
+
+[Available models](https://dmap-data-commons-ord.s3.amazonaws.com/index.html?prefix=#USEEIO-State/) - All states 2012-2020.
```
Comment (bl-young) [inline import_factors_exio/README.md:62]: These state models aren't related to this particular effort so this line should be removed

File: import_factors_exio/README.md [hunk 3]
```diff
@@ -1,5 +1,71 @@
...
+
+Now run the app. Before running, you can change the year range at the top of generate_import_factors.py. If the year is not yet available, the other years will still be generated. (2023 was not yet available on May 19, 2024.)
```
Comment (bl-young) [inline import_factors_exio/README.md:64]: I will likely adjust this text to specifically state what years have been tested.

**Review by bl-young — CHANGES_REQUESTED** [review]
as we discussed on email, let's re-do this PR without the data for 2017, 2018, 2020-2022 committed (probably no need to commit the 2019 results either since no changes were made outside of rounding).

so really the only thing that needs to be added is the updated readme

File: import_factors_exio/README.md [hunk 4]
```diff
@@ -1,5 +1,71 @@
...
+
+You can optionally create a virtual environment first and install the following python libraries:
+<!-- Let us know if you also need to add `pip install path` --->
+
+	python3 -m venv env
+	source env/bin/activate
+	pip install pandas
+	pip install pyyaml
+	pip install pymrio
+	pip install pathlib
```
Comment (bl-young) [inline import_factors_exio/README.md:25]: pathlib is a core python package so does not need to be installed with pip.

I'm also fine with supplying an env file here instead

**Comment by ModelEarth** [comment]
Reissuing with just Readme

**Pull request closed without merging**